src/reqstool/resources/schemas/v1/requirements.schema.json

```diff
@@ -250,11 +250,22 @@
                     "description": "Array of requirement categories. E.g. functional-suitability, usability, performance-efficiency, security, reliability, maintainability"
                 },
                 "references": {
-                    "type": "array",
-                    "items": {
-                        "$ref": "#/$defs/reference"
+                    "type": "object",
+                    "additionalProperties": false,
+                    "properties": {
+                        "requirement_ids": {
+                            "type": "array",
+                            "items": {
+                                "$ref": "common.schema.json#/$defs/reqid",
```

Comment:
is this a reqid or an urn_id or perhaps both? the def in common might need an update if it is in fact urn_id.

Reply:
I have just reused the definition that we used previously. 
The definition for reqid in the common schema is just any string, and there are currently no definition set for urnid.

 This field is always interpreted a string, and if there is a ":" in it, reqstool will split it up as a urn_id .

I don't know it that is robust enough. What would you propose as an alternate solution?

Reply:
Not sure. Let's write a ticket to follow-up on it. Or?

Reply:
yes, I'll do that!